sql/system.go

```diff
@@ -34,12 +36,10 @@ const (
 	RootNamespaceID ID = 0
 
 	// System IDs should remain <= MaxReservedDescID.
-	// systemDatabaseID is the ID of the system database.
-	systemDatabaseID ID = 1
-	// namespaceTableID is the ID of the namespace table.
-	namespaceTableID ID = 2
-	// descriptorTableID is the ID of the descriptor table.
```

Comment:
yes, it's terrible. But this is really the type of comment that provides no value.